src/Quiz/Quiz.js

```diff
@@ -67,21 +69,34 @@ const Quiz = (props) => {
     const modifyWeatherData = quizData.weather.map(weather => {
       return `%${weather}%`
     })
+
     const submissionData = {
       data: {
-        id: null,
+        id: 0,
```

Comment:
Do we want to hardcode this id as 0 or is this a testing thing? @alyssabull @jaypeasee

Reply:
Let's keep it as 0 for now!